ouroboros-consensus/docs/tutorials/Ouroboros/Consensus/Tutorial/Simple.lhs

```diff
@@ -578,6 +579,9 @@ If it did, that means the `LedgerState` is such that it is in a state that will
 eventually fail due to the passage of time and such errors should have been
 signalled earlier (for example, when applying blocks.)
```

Comment:
In this sentence, should it not say "Note that this function _does not allow failure_"?